src/Umbraco.PublishedCache.HybridCache/DatabaseCacheRebuilder.cs

```diff
@@ -21,4 +40,25 @@ public void Rebuild()
         _databaseCacheRepository.Rebuild();
         scope.Complete();
     }
+
+    public void RebuildDatabaseCacheIfSerializerChanged()
+    {
+        NuCacheSerializerType serializer = _nucacheSettings.Value.NuCacheSerializerType;
```

Comment:
The entire method needs to be in one scope. I managed to get the database key-value updated when the rest failed. Thereby it will not update.. 

Ill add it